Scaffold analyzer (#86)
ionide/ionide-analyzers: docs/content/contributions.md

### Your analyzer

Scaffold your analyzer by running:

```shell
dotnet fsi .\build.fsx -- -p NewAnalyzer
```

> This will ask prompt your to enter a name and a category and create will all the necessary files.

Reviewer: ```suggestion
This will prompt you to enter the analyzer's name and a category and will create all the necessary files.
```
